src/api/app/models/package.rb

```diff
@@ -878,8 +878,9 @@ def build_result_old(repository, view = [])
     Buildresult.find(project: project, package: self, repository: repository, view: view)
   end
 
-  def buildresults
-    results = Buildresult.find_hashed(project: project.name, package: name, view: 'status', multibuild: '1', locallink: '1')
+  def buildresults(prj = nil)
+    prj = project unless prj
```

Comment:
Why? : confused:

The [Ruby style guide](https://github.com/bbatsov/ruby-style-guide/blob/master/README.md#spaces-around-equals) says:

> Use spaces around the `=` operator when assigning default values to method parameters


Although for example _pep8_, the style analizer for Python, likes just the opposite